specification/cognitiveservices/data-plane/Language/preview/2023-11-15-preview/analyzedocuments.json

```diff
@@ -0,0 +1,2237 @@
+            "value": "USSocialSecurityNumber",
+            "description": "US Social Security Number"
+          },
+          {
+            "name": "UAPassportNumberDomestic",
+            "value": "UAPassportNumberDomestic",
+            "description": "UA Passport Number Domestic"
+          },
+          {
+            "name": "UAPassportNumberInternational",
+            "value": "UAPassportNumberInternational",
+            "description": "UA Passport Number International"
+          },
+          {
+            "name": "Organization",
+            "value": "Organization",
+            "description": "Organization"
+          },
+          {
+            "name": "Email",
+            "value": "Email",
+            "description": "Email"
+          },
+          {
+            "name": "URL",
+            "value": "URL",
+            "description": "URL"
+          },
+          {
+            "name": "Age",
+            "value": "Age",
+            "description": "Age"
+          },
+          {
+            "name": "PhoneNumber",
+            "value": "PhoneNumber",
+            "description": "Phone Number"
+          },
+          {
+            "name": "IPAddress",
+            "value": "IPAddress",
+            "description": "IP Address"
+          },
+          {
+            "name": "Date",
+            "value": "Date",
+            "description": "Date"
+          },
+          {
+            "name": "Person",
+            "value": "Person",
+            "description": "Person"
+          },
+          {
+            "name": "Address",
+            "value": "Address",
+            "description": "Address"
+          },
+          {
+            "name": "All",
+            "value": "All",
+            "description": "All PII categories."
+          },
+          {
+            "name": "Default",
+            "value": "Default",
+            "description": "Default PII categories for the language."
+          }
+        ]
+      }
+    },
+    "PiiTaskParameters": {
+      "type": "object",
+      "description": "Supported parameters for a PII Entities Recognition task.",
+      "properties": {
+        "domain": {
+          "$ref": "#/definitions/PiiDomain"
+        },
+        "piiCategories": {
+          "$ref": "#/definitions/PiiCategory"
+        },
+        "excludePiiCategories": {
+          "$ref": "#/definitions/PiiCategoriesExclude"
+        },
+        "stringIndexType": {
+          "$ref": "common.json#/definitions/StringIndexType"
+        },
+        "redactionCharacter": {
+          "$ref": "#/definitions/RedactionCharacter"
+        }
+      },
+      "allOf": [
+        {
+          "$ref": "common.json#/definitions/PreBuiltTaskParameters"
+        }
+      ]
+    },
+    "PiiDomain": {
+      "type": "string",
+      "description": "The PII domain used for PII Entity Recognition.",
+      "default": "none",
+      "enum": [
+        "phi",
+        "none"
+      ],
```

Comment:
You should be consistent in your casing for enums. You use both PascalCase and camelCase in this document.

Reply:
Currently keeping consistent with naming in analyzetext (naming is similar for the same enums). We will have a client.tsp for casing corrections that will make it more convenient for SDK release.